src/Raygun.NetCore.Blazor/Models/EnvironmentDetails.cs

```diff
@@ -191,7 +191,7 @@ internal EnvironmentDetails(BrowserSpecs specs, BrowserStats stats)
             DeviceName = !string.IsNullOrWhiteSpace(specs.DeviceName) ? specs.DeviceName : null; ;
             Locale = specs.Locale;
             OSVersion = !string.IsNullOrWhiteSpace(specs.UAHints?.CalculatedOSVersion) ? specs.UAHints.CalculatedOSVersion : specs.CalculatedOSVersion;
-            Platform = specs.UAHints.CalculatedPlatform ?? specs.Platform;
+            Platform = specs.UAHints?.CalculatedPlatform ?? specs.Platform;
```

Comment:
The IDE also displayed a warning here saying that `UAHints` could be null. One of the tests failed due to it being null too.